lib/editor/components/timetable/TimetableEditor.js

```diff
@@ -216,7 +216,7 @@ export default class TimetableEditor extends Component<Props, State> {
         }
         // }
         // if (stop.timepoint === null || stop.timepoint) {
```

Comment:
I know these are unrelated, but do you think we can remove these comments?

Reply:
Git blame says they're from 4 years ago. I would but this file is difficult enough to parse as it is I think removing the historical record of what happened here might hurt more than it helps